mesos-rxjava-example/mesos-rxjava-example-framework/src/main/java/org/apache/mesos/rx/java/example/framework/sleepy/Sleepy.java

```diff
@@ -203,7 +219,7 @@ private static TaskInfo sleepTask(@NotNull final AgentID agentId, @NotNull final
                     .setEnvironment(Environment.newBuilder()
                         .addVariables(
                             Environment.Variable.newBuilder()
-                                .setName("SLEEP_SECONDS").setValue("15")
+                                .setName("SLEEP_SECONDS").setValue(sleepSeconds)
```

Comment:
There are parts of this file that are duplicated in `LegacySleepy`. If we want to keep them in sync, does it make sense to factor out the common pieces?

Either that, or we should have a test suite that can verify they are behaving in the same way.